jenkins/hourly_maintenance.py

```diff
@@ -194,6 +196,29 @@ def DeleteCorruptGitRepos():
     return err
 
 
+def DeleteOldWorkspaces():
+    err = 0
+    live_jobs = None
+    for host in ('jenkins-master', 'pull-jenkins-master'):
```

Comment:
Yeah, but then I have to also match against master... just trying both seems fine and easy?